src/playground/sample_data_tooling/sample_data_generator/README.md

```diff
@@ -0,0 +1,187 @@
+# Sample Data Tooling: Generation
+
+Before using this tool, make sure the required libraries as listed in `requirements.txt` are downloaded. As this tool only generates data entries based on the [Faker library](https://faker.readthedocs.io), running this tool alone does not require configuration of OpenSearch (OS) or OpenSearch Dashboards (OSD). 
+
+Additionally, this tool requires some familiarization of Faker methods and optional arguments to both use this tool and find all supported data types.
+
+## Methods
+
+- `choose_field()`: Given a data type, this function will generate a single data value using Faker
+    - **Arguments:**
+        - `kind` (string): The type of variable to be generated
+            - *ex:* `choose_field("ascii_company_email")` taken directly from [ascii_company_email](https://faker.readthedocs.io/en/master/providers/faker.providers.internet.html#faker.providers.internet.Provider.ascii_company_email)
+            ~~~
+            >>> choose_field("ascii_company_email")
+            'achang@green.info'
+            ~~~
+            - *ex:* `choose_field("job")` taken directly from [job](https://faker.readthedocs.io/en/master/providers/faker.providers.job.html)
+            ~~~
+            >>> choose_field("job")
+            'Musician'
+            ~~~
+            - *ex:* `choose_field("float")` taken from `options/custom-field-types.json`
+            ~~~
+            >>> choose_field("float")
+            '-69804.3874773259'
+            ~~~
+        ### Quick Notes on "kind" Argument
```

Comment:
can we put "Quick Notes on "kind" Argument" a little small, and it should be belonged to kind (string): , right ?

Reply:
> can we put "Quick Notes on "kind" Argument" a little small

Done, it's now a bolded bulleted list

> should be belonged to kind (string): , right

Yes, I modified the line to "`kind` (string)"